### ISSUE TITLE
Update to the new protocol logo: moz://a

### ISSUE THREAD

**Issue by pmac** [mozilla/tabzilla#84]
We should hold off on doing this until the font is ready so we can potentially use that. But I thought having an issue here to track it would be good.

/cc @craigcook @schalkneethling @bensternthal

**Comment by fitojb** [comment]
The SVG monochrome logo with transparent letters can be downloaded from https://mozilla.ninja/s/Mozilla_Large-Logo_One-color-white.zip . IMHO the font is not a blocker

**Comment by craigcook** [comment]
At the time this issue was created we knew very little about the new branding and were under the impression we were supposed to use a font to render the logo as live text. Once we learned more and a first draft of the brand guidelines became available that turned out not to be the case. 

Which is good, because we should use an image instead of a font anyway. Webfonts are blocked by some devices and/or ad blockers, and can be overridden by user preferences. An image is a more reliable way to render the logo consistently. People can also block images, of course, but from a brand perspective displaying no logo is probably better than displaying the wrong logo.

**Comment by alexgibson** [comment]
Tabzilla is deprecated, closing this as wontfix.
